rest/route/github.go

```diff
@@ -305,11 +307,11 @@ func (gh *githubHookApi) retryPRPatch(ctx context.Context, owner, repo string, p
 		return errors.Wrapf(err, "can't get PR for repo %s:%s, PR #%d", owner, repo, prNumber)
 	}
 
-	return gh.AddIntentForPR(pr)
+	return gh.AddIntentForPR(pr, initiator)
 }
 
-func (gh *githubHookApi) AddIntentForPR(pr *github.PullRequest) error {
-	ghi, err := patch.NewGithubIntent(gh.msgID, pr)
+func (gh *githubHookApi) AddIntentForPR(pr *github.PullRequest, owner string) error {
```

Comment:
Can these functions use consistent terminology? (i.e. owner or initiator)

Reply:
They're consistent for what they know about. This function is just told the owner of the patch, but its callers know the initiator of the event